Tests/xSharePoint/xSharePoint.xSPDesignerSettings.Tests.ps1

```diff
@@ -123,14 +125,54 @@ Describe "xSPDesignerSettings" {
                 Test-TargetResource @testParams | Should Be $false
             }
 
-            $Global:xSharePointDesignerUpdated = $false
             It "updates the SharePoint Designer settings" {
                 Set-TargetResource @testParams
-                $Global:xSharePointDesignerUpdated | Should Be $true
+                Assert-MockCalled Get-SPSite
```

Comment:
Why are we checking for Get-SPSite to be called here? isn't the update call the critical one?